js/main.js

```diff
@@ -23,5 +23,107 @@ function addingInputIntoObject(event) {
+        $allViews[eachView].classList.replace('hidden', 'view');
+        data.view = event.target.getAttribute('data-view');
+      }
+    }
+  }
+}
+$form.addEventListener('submit', swappingViews);
+
+var $anchor = document.querySelector('.anchor');
+function anchor(event) {
+  if (event.target.tagName === 'A') {
+    var $dataView = event.target.getAttribute('data-view');
+    for (var eachView = 0; eachView < $allViews.length; eachView++) {
+      $allViews[eachView].className = 'view hidden';
+      if ($dataView === $allViews[eachView].getAttribute('data-view')) {
+        $allViews[eachView].classList.replace('hidden', 'view');
+        data.view = event.target.getAttribute('data-view');
+      }
+      if ($allViews[eachView].matches('#navigation')) {
+        $allViews[eachView].classList.remove('hidden');
+      }
+    }
+  }
+}
+$anchor.addEventListener('click', anchor);
+
+function generateEntryDomTree(entries) {
+  var $li = document.createElement('li');
+
+  var $divRow = document.createElement('div');
+  $divRow.setAttribute('class', 'row');
+  $li.appendChild($divRow);
+
+  var $divHalfColumn = document.createElement('div');
+  $divHalfColumn.setAttribute('class', 'column-half');
+  $divRow.appendChild($divHalfColumn);
+
+  var $divHalfColumn2 = document.createElement('div');
+  $divHalfColumn2.setAttribute('class', 'column-half');
+  $divRow.appendChild($divHalfColumn2);
+
+  var $divImageContainer = document.createElement('div');
+  $divImageContainer.setAttribute('class', 'image-container');
+  $divHalfColumn.appendChild($divImageContainer);
+
+  var $img = document.createElement('img');
+  $img.setAttribute('class', 'image-size');
+  $img.setAttribute('src', entries.photoURL);
+  $divImageContainer.appendChild($img);
+
+  var $heading = document.createElement('h3');
+  var $headingText = document.createTextNode(entries.title);
+  $heading.appendChild($headingText);
+  $divHalfColumn2.appendChild($heading);
+
+  var $paragraph = document.createElement('p');
+  var $paragraphText = document.createTextNode(entries.notes);
+  $paragraph.appendChild($paragraphText);
+  $divHalfColumn2.appendChild($paragraph);
+
+  return $li;
+}
+
+function appendTheDom(event) {
+  var $ul = document.querySelector('.list');
+  for (var i = 0; i < data.entries.length; i++) {
+    var $newEntry = generateEntryDomTree(data.entries[i]);
+    $ul.appendChild($newEntry);
+  }
+}
+document.addEventListener('DOMContentLoaded', appendTheDom);
+
+function previousDataView(event) {
+  var $view;
+  $view = data.view;
+  for (var eachView = 0; eachView < $allViews.length; eachView++) {
+    $allViews[eachView].className = 'hidden';
+    if ($view === $allViews[eachView].getAttribute('data-view')) {
+      $allViews[eachView].classList.replace('hidden', 'view');
+    }
+    if ($allViews[eachView].matches('#navigation')) {
+      $allViews[eachView].classList.remove('hidden');
+    }
+  }
+}
+previousDataView(Event);
```

Comment:
What's the purpose of the argument here? You don't seem to use the parameter in the function definition at all.